Makefile.am

```diff
@@ -813,6 +813,18 @@ endif
 
 check: $(CHECK_DEPS)
 
+install-all: install-kernel |  install-image |  install
+
+# Get clear containers image path
+IMAGE_PATH := $(shell dirname $(CONTAINERS_IMG))
+install-image:
+	$(srcdir)/installation/install_clear_image.sh $(CLEAR_VM_IMAGE_VERSION) $(IMAGE_PATH)
```

Comment:
It isn't handled at all in CC 3.0 yet as the `Makefile` currently only installs the runtime binary and config file.